doc/developer/bestPractices/README

```diff
@@ -46,5 +46,8 @@ CHPL_DEVELOPER.txt
 TestSystem.rst
   All about the Chapel testing system.
 
+SpellChecking.rst
+  How to use chplspell to spellcheck the documentation, modules, and compiler.
```

Comment:
I had to check, but you do run this over the runtime.  Not listing it when other components are listed implies that you don't include it when that isn't the case.  Would you mind either expanding this list to be complete, or explicitly mentioning that it isn't?